src/app/core/shared/collection.model.ts

```diff
@@ -130,6 +130,14 @@ export class Collection extends DSpaceObject implements ChildHALResource, Handle
     return this.firstMetadataValue('dc.description.tableofcontents');
   }
 
+  /**
+   * The thumbail description of this Collection
+   * Corresponds to the metadata field dspace.thumbnail.description
+   */
+  get descriptionThumbnail(): string {
+    return this.firstMetadataValue('dspace.thumbnail.description');
```

Comment:
It was suggested in today's meeting that the more appropriate place to store the "alt" text for a Bitstream would be on the Bitstream object itself in the `dc.description` field.  This has two main benefits:
* It would avoid needing to create a new metadata field on the backend.
* It'd also be better to store this information on the Bitstream as the "alt" text is a description of the Bitstream, and not something that belongs to the Collection.  That way, if the Bitstream logo changes, then the old "alt" text goes away and new text comes with the new Bitstream.